storage/rocksdb/rdb_native_dd.cc

```diff
@@ -34,7 +34,10 @@ bool native_dd::is_dd_table_id(dd::Object_id id) {
 }
 
 int native_dd::reject_if_dd_table(const dd::Table *table_def) {
-  if (table_def != nullptr && is_dd_table_id(table_def->se_private_id())) {
```

Comment:
Same comments as for the parallel change in InnoDB (except for `innobase_is_ddse`)